web/src/lib/utils/helpers.ts

```diff
@@ -97,7 +97,8 @@ export const scrollToTop = (props?: { smooth?: boolean }): void => {
 
 export const scrollToTopOfElement = (
   element: HTMLDivElement | null,
-  { isDesktop }: { isDesktop: boolean }
+  { isDesktop }: { isDesktop: boolean },
+  waitTime: number | undefined = 100
```

Comment:
[sand] Cool idea about passing the `waitTime` as a param here! You could take this a step further and add it to the object we have defined on line 100.

Reply:
It could look something like this:
```js
export const scrollToTopOfElement = (
  element: HTMLDivElement | null,
  { isDesktop, waitTime = 100 }: { isDesktop: boolean; waitTime: number | undefined }
): void => {
```